bepasty/utils/permissions.py

```diff
@@ -54,7 +54,7 @@ def get_permissions():
 def get_permission_icons():
     return [
         (permission, permission_icons[permission])
-        for permission in get_permissions()
+        for permission in get_permissions() if permission
```

Comment:
no, your code is ok I guess. didn't try, but I hope you did this time.

i was just referring to some other code that needs to get optimized so that such "if permission" code is not needed, because the set only contains valid members.